buf/registry/module/v1beta1/label_service.proto

```diff
@@ -157,12 +163,22 @@ message ListLabelHistoryRequest {
 }
 
 message ListLabelHistoryResponse {
+  message Value {
+    // The Commit.
+    Commit commit = 1 [(buf.validate.field).required = true];
+
+    // The GovernanceWorkflowState for this Commit on this Label.
+    //
+    // This field will only be set if governance workflow was enabled on this Label when this Commit was pushed.
```

Comment:
What about blocked commits in a Label that doesn't have governance workflow enabled?

Reply:
updated docs to clarify, ptal